articles/azure/azs-identify-global-admins.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to identify your Global Admins| UKCloud Ltd
+description: How to identify your Azure Global Admins for Microsoft Azure
+services: azure-stack
+author: Paul Brown
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Identify your Global Admins
+toc_fullpath: Users/How To/azs-identify-global-admins.md
+toc_mdlink: azs-identify-global-admins.md
```

Comment:
as above (and remember to change the actual file name!)

Reply:
done